platforms/gke/base/core/deploy.sh

```diff
@@ -17,6 +17,10 @@ set -e
 
```

Comment:
In order to make this script more robust, consider setting the following options as well (in addition to `set -e` / `set -o errexit`):

`set -o nounset`
`set -o pipefail`

Reply:
(if we go this route, it's probably worth doing this in the `teardown.sh` script as well.)

Reply:
Considering that these scripts depend on environment variables being set, `nounset` makes sense in my opinion. The alternative would be to explicitly check if env variables are set, but that becomes not scalable pretty fast.